### PR TITLE
Minimal configuration example

### PR DESCRIPTION
**Pull request by fridgerator**

### CODE REVIEW

File: getting-started/Installation/minimal-configuration.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Minimal Configuration
+
+Amber can be run from a single file for mimal configuration setups if the user prefers this for smaller applications.
```
Comment (faustinoaq) [inline getting-started/Installation/minimal-configuration.md]: Typo `minal -> minimal`

**Comment by eliasjpr** [comment]
Did we write the techempower benchmark like this? We probably should've  :)

File: getting-started/Installation/minimal-configuration.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Amber::Server.configure do |app|
+  pipeline :api do
+  end
+
+  routes :api do
+    get "/", HelloController, :index
+  end
+end
+
+Amber::Server.instance.run
```
Comment (faustinoaq) [inline getting-started/Installation/minimal-configuration.md]: Isn't `Amber::Server.start` ?

Comment (eliasjpr) [reply getting-started/Installation/minimal-configuration.md]: Amber::Server.run

Comment (faustinoaq) [reply getting-started/Installation/minimal-configuration.md]: @eliasjpr Well, our default app template is using `.start` https://github.com/amberframework/amber/blob/master/src/amber/cli/templates/app/src/%7B%7Bname%7D%7D.cr#L3

**Pull request merged**